src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ManagedProjectSystemPackage.cs

```diff
@@ -50,6 +53,12 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
         {
```

Comment:
This method has become a bit of mess and will cleanup at after I've merged all these packages together.